pkg/builder/cbuildidx/builder.go

```diff
@@ -212,6 +212,10 @@ func (b CbuildIdxBuilder) Build() error {
 		args = append(args, "--target", b.BuildContext+"-database")
 	}
 
+	if b.Options.Generator == "Ninja" && !(b.Options.Debug || b.Options.Verbose) {
+		args = append(args, "--", "--quiet")
```

Comment:
@brondani Does this change mean
`args = append(args, "--quiet")`

Reply:
No, it means `-- --quiet` is added to `args`.
So `--quiet` is passed to the underlying build system, in this case Ninja, for example:
```
cmake --build <directory> --target <target> -- --quiet
```